src/test/cypress/cypress/integration/Monitoring.spec.js

```diff
@@ -107,4 +108,289 @@ describe ('Monitoring screen tests',function () {
         cy.loadTestConf();
         cy.reload();
     })
+
+
+
+    it('Check monitoring cards reception', function () {
+
+        cy.loginOpFab('operator1','test');
+        cy.send6TestCards();
+
+        // We move to archives screen
+        cy.get('#opfab-navbar-menu-monitoring').click();
+
+        // operator1 should see the grid of monitoring cards
+        cy.get("#opfab-monitoring-table").find('#opfab-monitoring-table-grid').should("exist");
+
+        // No card detail is displayed
+        cy.get('of-card-details').should('not.exist');
+
+        // Should have the correct number of cards (5 because one card is set not to be visible in monitoring)
+        cy.get('#opfab-monitoring-table-grid').find('.ag-center-cols-container').find('.ag-row').should('have.length', 5);
+
+        // Opens the first card, checks that its content is visible
+        cy.get('#opfab-monitoring-table-grid').find('.ag-center-cols-container').first().click();
+        cy.get('of-card-details').first().should('exist');
+
+        // Closes the card content and check card  body is no longer visible
+        cy.get("#opfab-close-card").click();
+        cy.get('of-card-details').should('not.exist');
+    })
+    
+
+    it('Check filters and reset button', function () {
+
+        cy.loginOpFab('operator1','test');
+        cy.send6TestCards();
+
+        // Access monitoring screen
+        cy.get('#opfab-navbar-menu-monitoring').click();
+
+        // Should have the correct number of cards (5 because one card is set not to be visible in monitoring)
+        cy.get('#opfab-monitoring-table-grid').find('.ag-center-cols-container').find('.ag-row').should('have.length', 5);
+
+        // Filter on IN PROGRESS cards
+        cy.get('#opfab-typeOfState').click();
+        cy.get('#opfab-typeOfState').contains('IN PROGRESS').should('exist');
+        cy.get('#opfab-typeOfState').find('.lazyContainer > li').first().click();
```

Comment:
Why not using a select by value , like cy.get(#opfab-typeOfState).select(xxx).

https://docs.cypress.io/api/commands/select#Value

Reply:
there is no "select" in the html so cy.select(...) will not work here